src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,14 @@
+@import "variables";
+
```

Comment:
please fix vertical centering and adjust day tiles to match gridlines
![Snapshot 2022-03-31 23 01 51](https://user-images.githubusercontent.com/39308717/161140320-0f2c661a-0914-4694-a656-1adbd3c02f41.png)